### DISCUSSION TITLE
Got banned without any warning, I desire my account back

### DISCUSSION THREAD

**Discussion by ghost** [community/community#88713] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

My ticket number is: 2528922

I contacted you last weekend but still haven't received any further contact from you.

My account, github.com/mishka, got banned without any warning. I didn't have a chance to back up any of my repos.

I don't think people wake up and think about getting banned from GitHub just like that.

Here it says: "Reinstatement: Where a user can agree to abide by our Acceptable Use Policies moving forward and has made the changes necessary to address the violation(s), we may choose to reinstate their account or content depending on the circumstances and severity of the initial violation."

https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement

I don't know what I've done, but I am keen on collaborating on the issue and removing/making changes accordingly to get my account back.

I use it for LinkedIn and many job applications and would like to have it back. This is a significant matter.

Please get in touch with me about this as soon as you can because I feel ignored now.

Kind Regards,

**Comment by queenofcorgis** [comment, marked as answer]
Hi @gotbannedfornoreason,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
